package.json

```diff
@@ -44,7 +44,7 @@
     "serve": "*",
     "sinon": "1.17.7",
     "unexpected": "10.23.0",
-    "unexpected-documentation-site-generator": "^4.2.3",
-    "unexpected-markdown": "1.4.1"
+    "unexpected-documentation-site-generator": "4.5.0",
```

Comment:
But no worries, it is fine. I did it while I was doing a lot of updates to the documentation generators.